theatre/studio/src/uiComponents/simpleContextMenu/RightClickMenu/RightClickMenu.tsx

```diff
@@ -80,7 +81,7 @@ const RightClickMenu: React.FC<{
 
     const onMouseMove = (e: MouseEvent) => {
       if (
-        e.clientX < pos.left - pointerDistanceThreshold ||
+        (!props.keepOpen && e.clientX < pos.left - pointerDistanceThreshold) ||
```

Comment:
Suggest: I strongly recommend trying to use another early return style here for each case. And generally cleaning this up, because it looks very strange that `props.keepOpen` is only affecting one part of the `||` expression chain.

Reply:
this code was dead and the change is removed.